project/SotaBuild.scala

```diff
@@ -89,6 +91,18 @@ object SotaBuild extends Build {
       testOptions in RandomTests += Tests.Argument(TestFrameworks.ScalaTest, "-n", "RandomTest"),
       parallelExecution in Test := true,
       dockerExposedPorts := Seq(8081),
+      dockerCommands := dockerCommands.value.filterNot {
```

Comment:
Ok, I made those changes. The entrypoint.sh script now behaves gracefully if DB_ALIVE_URL isn't specified, so the changes should no longer have any potential to break anything.

Also, I tracked down the reason it was intermittently failing on teamcity: the resolver random tests job didn't have a cleanup step. If the most recent job run on a particular agent was ci_test, my build would run fine because it had been cleaned up. If resolver_random was the most recent job run on the agent I used, it would fail trying to bind port 3306 (because there was still a docker container running, left over from that job).